integration_tests/mockApis/ciagInducationApi.ts

```diff
@@ -128,4 +128,114 @@ const stubGetCiagProfile500Error = (prisonNumber = 'G6115VJ'): SuperAgentRequest
     },
   })
 
-export default { stubGetCiagProfile, stubGetCiagProfile404Error, stubGetCiagProfile500Error }
+const stubCiagInductionList = (): SuperAgentRequest =>
```

Comment:
I thought this was the most appropriate file for this to go in

Reply:
Yes, thats a perfect choice of file 👍